lib/SIL/ValueOwnership.cpp

```diff
@@ -284,6 +284,11 @@ ValueOwnershipKind ValueOwnershipKindClassifier::visitBeginApplyResult(
   return Result->getOwnershipKind();
 }
 
+ValueOwnershipKind ValueOwnershipKindClassifier::visitCXXVirtualMethodResult(
+    CXXVirtualMethodResult *Result) {
+  return Result->getOwnershipKind();
```

Comment:
Surely there must be a better way to do this. @gottesmm?

Reply:
We should be able to for stuff like this use a CXX macro since the code is exactly the same.